persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java

```diff
@@ -48,6 +48,8 @@ private PersistToolsConstants() {
     public static final String DATABASE_PLACEHOLDER = "${%s.clients.database}";
 
     public static final String CONFIG_SCRIPT_FILE = "Config.toml";
+    public static final String PERSIST_TOML_FILE = "Persist.toml";
+    public static final String BALLERINA_SCRIP_FILE = "Ballerina.toml";
```

Comment:
```suggestion
    public static final String BALLERINA_TOML_FILE = "Ballerina.toml";
```

Reply:
We are removing this as this was created by overlapping two PRs

Reply:
removed in [36fe5f0](https://github.com/ballerina-platform/persist-tools/pull/54/commits/36fe5f0b814d539d7d6bdb0bbf09e3dbc175efc6)